.idea/encodings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file specific to your IDE/dev env?  If so, I don't think it needs to be checked-in

Reply:
This is a folder created by Intellij IDEA. I would propose to add it to .gitignore of the module similar to Eclipse specific folders/files.
@craigmcnally is it acceptable?

Reply:
yes, sounds good

Reply:
Fixed. Thanks.